app/src/main/java/org/y20k/trackbook/Keys.kt

```diff
@@ -33,9 +33,9 @@ object Keys {
     const val CURRENT_TRACKLIST_FORMAT_VERSION: Int = 0
 
     // intent actions
-    const val ACTION_START: String = "org.y20k.trackbooks.action.START"
-    const val ACTION_STOP: String = "org.y20k.trackbooks.action.STOP"
-    const val ACTION_RESUME: String = "org.y20k.transistors.action.RESUME"
+    const val ACTION_START: String = "org.y20k.trackbook.action.START"
+    const val ACTION_STOP: String = "org.y20k.trackbook.action.STOP"
```

Comment:
The extra `s` was not a problem, since all places use these constants. But I figured that I should clean them up anyway to make them consistent.